VERSIONING.md

```diff
@@ -0,0 +1,109 @@
+# oneTBB Versioning Policies
+
+There are several types of "versioning" used by the oneTBB project.
+
+1. The *oneTBB specification version*
+2. The *official oneTBB library version*
+3. The *library interface version*
+4. *API versioning* used to maintain backwards compatibility
+
+The oneTBB project uses semantic versioning (https://semver.org/) and as a oneAPI Library,
+the oneTBB project makes commitments around compatibility as described
+[here](https://www.intel.com/content/www/us/en/docs/oneapi/programming-guide/2024-1/oneapi-library-compatibility.html).
+With semantic versioning, a MAJOR version increment generally means an incompatible API change,
+while a MINOR version increment means added functionality that is implemented in a backwards compatible manner.
+
+## The oneTBB Specification Version
+
+The oneTBB project is governed by the [UXL foundation](https://uxlfoundation.org/) and has
+[an open specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onetbb/source/nested-index).
+Each release of the specification has a version number.
+
+[As described in the specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onetbb/source/configuration/version_information),
+the value of the `ONETBB_SPEC_VERSION` macro in `oneapi/tbb/version.h` is  latest specification of oneTBB fully
+supported by the implementation.
+
+## The oneTBB Library Version
+
+The oneTBB library version can be thought of as the name of the release. If you want to find a specific
+release in the repository (https://github.com/uxlfoundation/oneTBB), the tag you would look for corresponds
+to the oneTBB library version.
+
+[As described in the specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onetbb/source/configuration/version_information),
+the macros `TBB_VERSION_MAJOR`, `TBB_VERSION_MINOR`, `TBB_VERSION_STRING` in `oneapi/tbb/version.h`
```

Comment:
There is also TBB_VERSION_PATCH and __TBB_VERSION_SUFFIX (internal). Shouldn't they be mentioned? (or at least TBB_VERSION_PATCH because it is a part of official version) but usage of __TBB_VERSION_SUFFIX is very rare and specific.)